contrib/gcppubsub/pkg/reconciler/gcppubsubsource.go

```diff
@@ -41,9 +46,41 @@ import (
 )
 
 const (
+	// controllerAgentName is the string used by this controller to identify
+	// itself when creating events.
+	controllerAgentName = "gcp-pubsub-source-controller"
+
+	// raImageEnvVar is the name of the environment variable that contains the receive adapter's
+	// image. It must be defined.
+	raImageEnvVar = "GCPPUBSUB_RA_IMAGE"
+
 	finalizerName = controllerAgentName
 )
 
+// Add creates a new GcpPubSubSource Controller and adds it to the Manager with
+// default RBAC. The Manager will set fields on the Controller and Start it when
```

Comment:
What does 'default RBAC' mean?

Reply:
what fields will Manager set on the Controller? Maybe a pointer to them?

Reply:
Add clarity to the comment for gcppubsubsource.Add()

Reply:
https://github.com/knative/eventing-sources/issues/202